falcon/errors.py

```diff
@@ -49,18 +49,17 @@ class HTTPUnauthorized(HTTPError):
         title (str): Error title (e.g., 'Authentication Required').
         description (str): Human-friendly description of the error, along with
             a helpful suggestion or two.
-        scheme (str): Authentication scheme to use as the value of the
-            WWW-Authenticate header in the response (default ``None``).
+        challenges (iterable of strings): Authentication scheme to use as the
```

Comment:
"Authentication scheme" => "One or more authentication challenges to use as the value of the..."